ethr-did/src/main/java/me/uport/sdk/ethrdid/EthrDID.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("UndocumentedPublicFunction", "UndocumentedPublicClass")
```

Comment:
Is this class deprecated? 
If no, let's add comments as well

Reply:
It is not deprecated but not intended for public use either.

After 1.0 release, this should be publicized properly (properly tested, comments added, use case demoed in demoapp, and documented in /docs)

Reply:
Alright.